minifi-toolkit/minifi-toolkit-configuration/src/test/resources/StressTestFramework.yml

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+MiNiFi Config Version: 2
```

Comment:
Will do, ended up using version 1 for this one though so that ids are optional as long as names are unique.